webpack.config.js

```diff
@@ -25,14 +25,18 @@ const transpileNodeModules = [
 const config = {
   entry: resolve('./src/index.js'),
   output: {
-    path: path.resolve(__dirname, 'dist'),
+    path: path.resolve(__dirname, "dist"),
```

Comment:
Nitpicky suggestion:

```suggestion
    path: path.resolve(__dirname, 'dist'),
```